response.go

```diff
@@ -13,6 +13,48 @@ import (
 	"os"
 )
 
+// Error codes adapted from go-ipfs-cmds. We should find a better solution.
+
+// ErrorType signfies a category of errors
+type ErrorType uint
```

Comment:
@magik6k ideas where to put this?

Reply:
I'm assuming that this is mirrored from go-ipfs-cmds - if that's the case I'd just import cmds here as it really what is implemented here

Reply:
Done. I'd rather not depend on that library in the long run but that's probably the best solution for now.